manage.py

```diff
@@ -19,11 +19,11 @@
         # exceptions on Python 2.
         try:
             import django  # pylint: disable=unused-import
-        except ImportError:
+        except ImportError as import_exception:
             raise ImportError(
                 "Couldn't import Django. Are you sure it's installed and "
                 "available on your PYTHONPATH environment variable? Did you "
                 "forget to activate a virtual environment?"
-            )
+            ) from import_exception
```

Comment:
Huh, I hadn't noticed this was added to Python 3.  Really need to go back and review all the release notes now that we're mostly on Python 3.8...